File: API/Helpers/UserParams.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Helpers
{
    public class UserParams : PaginationParams
    {

        public string CurrentUserName { get; set; }
        public string Faculty { get; set; } 
        public int minAge { get; set; } = 18;
        public int MaxAge { get; set; } = 60;

        public string OrderBy { get; set; } = "LastActive";
    }
}

File: client/src/app/_models/userParams.ts
import { HRUser } from "./user";

export class UserParams {
    faculty: string;
    mingAge = 18; 
    maxAge = 30;
    pageNumber = 1; 
    pageSize = 15; 
    orderBy = 'lastActive';

    constructor(user: HRUser) {
        this.faculty = user.faculty;
    }

}